app/api/sponsors.py

```diff
@@ -0,0 +1,70 @@
+from marshmallow_jsonapi import fields
+from marshmallow_jsonapi.flask import Schema, Relationship
+from flask_rest_jsonapi import ResourceDetail, ResourceList, ResourceRelationship
+
+from app.api.helpers.permissions import jwt_required
+from app.models import db
+from app.models.sponsor import Sponsor
+from app.models.event import Event
+
+
+class SponsorSchema(Schema):
+
+    class Meta:
+        type_ = 'sponsor'
+        self_view = 'v1.sponsor_detail'
+        self_view_kwargs = {'id': '<id>'}
+
+    id = fields.Str(dump_only=True)
+    name = fields.Str()
+    description = fields.Str()
+    url = fields.Str()
+    level = fields.Str()
+    logo_url = fields.Str()
+    sponsor_type = fields.Str()
+    event = Relationship(attribute='event',
+                         self_view='v1.sponsor_event',
+                         self_view_kwargs={'id': '<id>'},
+                         related_view='v1.event_detail',
+                         related_view_kwargs={'id': '<id>'},
+                         schema='SponsorSchema',
```

Comment:
Shouldn't this be `EventSchema` ? According to http://flask-rest-jsonapi.readthedocs.io/en/latest/quickstart.html#first-example, I think it should. Correct me if I am wrong.